source/NeutronaWand/Colours.h

```diff
@@ -235,3 +235,16 @@ CRGB getHueAsGRB(uint8_t i_colour, uint8_t i_brightness = 255) {
   // Forward to getHueAsRGB() with the flag set for GRB color swap.
   return getHueAsRGB(i_colour, i_brightness, true);
 }
+
+CRGB getHueColour(uint8_t i_colour, uint8_t i_num_leds, uint8_t i_brightness = 255) {
+  switch(i_num_leds) {
+    case 48:
```

Comment:
Could we instead use an ENUM for STOCK_BARREL and FRUTTO_BARREL just to avoid specific LED counts? I doubt we'll have many other configurations from those, and it gives a more user-friendly name for identification throughout. Asking as you're not really acting upon the count of the LED's but the type of LED's to change from GRB (stock) to RGB (Frutto).